keyboards/dz60/keymaps/danbee/keymap.c

```diff
@@ -0,0 +1,66 @@
+#include QMK_KEYBOARD_H
+
+// Name layers
+#define _QW 0
+#define _F1 1
+#define _F2 2
```

Comment:
Completely optional, but using an `enum` here makes this section easier to maintain.

```c
enum layer_names {
    _QW,
    _F1,
    _F2,
};
```

This is equivalent to what you have now once compiled.

Reply:
This is a good suggestion, thanks!